src/modal/modal.js

```diff
@@ -457,6 +457,12 @@ angular.module('ui.bootstrap.modal', ['ui.bootstrap.multiMap', 'ui.bootstrap.sta
             close: '$close($value)',
             dismiss: '$dismiss($value)'
           });
+          Array.prototype.forEach.call(
+            Object.keys(modal.scope.$resolve),
+            function(property) {
+              content.attr(property, '$resolve.' + property);
```

Comment:
Even better would be to call the angular-internal function for this conversion like angular.snakeCase() pseudo code....